### ISSUE TITLE
Junk Codes instead of delay

### ISSUE THREAD

**Issue by arash88** [GFW-knocker/gfw_resist_tls_proxy#36]
Hey @GFW-knocker 
Is it maybe possible to get rid of the delay mechanism and send Junk fragments between the valid ones?

**Comment by byehack** [comment]
Can dest server separate these junk fragments from valid ones?

**Comment by arash88** [comment]
> Can dest server separate these junk fragments from valid ones?

Yes, they will be dropped if these fragments are invalid in any sort of way like the wrong checksum

**Comment by byehack** [comment]
Tested now. it seems junk fragments also avoid packet to be complete, it prevents next fragments to be accepted by server:
`upstream: ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None)`

**Comment by arash88** [comment]
> Tested now. it seems junk fragments also avoid packet to be complete, it prevents next fragments to be accepted by server: `upstream: ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None)`

Please check this repo, it does that and a lot more:
https://github.com/ValdikSS/GoodbyeDPI

**Comment by byehack** [comment]
> https://github.com/ValdikSS/GoodbyeDPI/blob/master/src/fakepackets.c

it seems that using full packet junks. btw please check [here](https://github.com/byehack/gfw_resist_tls_proxy/compare/NoClass...byehack:gfw_resist_tls_proxy:junk) if you want junked `fragments`.

**Comment by byehack** [comment]
> https://github.com/ValdikSS/GoodbyeDPI/blob/master/src/fakepackets.c

it seems that using full packet junks. please check [here](https://github.com/byehack/gfw_resist_tls_proxy/compare/NoClass...byehack:gfw_resist_tls_proxy:junk) if you want test junked `fragments`.

**Comment by arash88** [comment]
> 

Yes, it's working just fine. In some cases like Youtube, you don't even need a v2ray connection!
